bindings/matrix-sdk-crypto-js/src/identities.rs

```diff
@@ -85,6 +85,28 @@ impl OwnUserIdentity {
 
         future_to_promise(async move { Ok(me.trusts_our_own_device().await?) })
     }
+
+    /// Get the master key of the identity.
+    #[wasm_bindgen(getter, js_name = "masterKey")]
+    pub fn master_key(&self) -> Option<String> {
+        let master_key = self.inner.master_key().as_ref().to_owned();
+        serde_json::to_string(&master_key).ok()
```

Comment:
Are we sure that we want to silently drop serialization issues here?

```suggestion
        serde_json::to_string(master_key).ok()
```

Reply:
Updated